### PR TITLE
Parallax browsers compatibility

### PR DESCRIPTION
**Pull request by leono286**

### CODE REVIEW

File: src/app/shared/directives/madness.directive.ts [hunk 1]
```diff
@@ -121,10 +128,12 @@ export class MadnessDirective implements AfterViewInit, OnChanges {
 
   @HostListener('window:mousemove', ['$event'])
   mousemoveEventHandler(event: any) {
-    if (
-      event.path.indexOf(this.element.nativeElement) !== -1 &&
-      this.madnessEnabled
-    ) {
+    if (event.path) {
+      var path = event.path;
```
Comment (jdjuan) [inline src/app/shared/directives/madness.directive.ts]: Use `const` or `let` next time

File: src/app/shared/directives/madness.directive.ts [hunk 2]
```diff
@@ -80,6 +80,13 @@ export class MadnessDirective implements AfterViewInit, OnChanges {
     });
   }
 
+  parallaxBoxInPath(elmnt):boolean{
+    while ((elmnt = elmnt.parentElement) && !elmnt.classList.contains('parallax-box')) {
```
Comment (jdjuan) [inline src/app/shared/directives/madness.directive.ts]: `elmnt` should have been `element`

Comment (leono286) [reply src/app/shared/directives/madness.directive.ts]: agree

File: src/app/shared/directives/madness.directive.ts (hunk 1 above)
Comment (leono286) [reply src/app/shared/directives/madness.directive.ts]: not agree

**Pull request merged**